lib/engines/content_object_store/test/factories/content_block.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :content_block, class: "ContentObjectStore::ContentBlock" do
```

Comment:
should we call this `content_block_document` to signify the behaviour we are expecting/imitating?

Reply:
Yeah, I was kinda iffy on that naming, but I think it makes the intention clearer

Reply:
cool, will rename `content_block` to `content_block_document`